constraintula/core.py

```diff
@@ -210,12 +212,22 @@ def get_functions(self) -> Mapping[Symbol, Callable]:
         return functions
 
 
-def solve(
-    independents: Mapping[str, Any], variables: Sequence[str], constraints: Iterable[Expr]
-) -> Mapping[str, Any]:
-    assignments = {Symbol(k): v for k, v in independents.items()}
-    values = System(constraints).with_independents(assignments).evaluate(assignments)
-    return {k: values[Symbol(k)] for k in variables}
+def collect_symbols(expr):
```

Comment:
Type annotations?

Reply:
I added annotations but I wasn't sure how to check them. What command do you use?

Reply:
Well, the simplest thing is to pip install mypy and just run it against the target .py file. We could set up a script to run against all source files.